x-pack/plugins/security_solution/server/lib/timeline/routes/timelines/get_timelines/index.ts

```diff
@@ -39,11 +43,12 @@ export const getTimelinesRoute = (
       },
     },
     async (context, request, response) => {
+      const customBadRequestError = new CustomBadRequestError('bad request error').get;
```

Comment:
Why do you hardcode the error message here? `throwErrors` is supposed to accept an error constructor to which an error message will be passed:

```ts
const createBadRequestError = (message: string) => new CustomBadRequestError(message);
// ...
fold(throwErrors(createBadRequestError), identity)
```

Reply:
done 7615a9f2cac7e8ef25f30b4a204d6c13c88313a7

Reply:
done 7615a9f2cac7e8ef25f30b4a204d6c13c88313a7